src/apps/bridge/bridgePowerController.cpp

```diff
@@ -227,6 +228,7 @@ void BridgePowerController::_update(
         if (nextSampleEpochS > currentCycleS) {
           powerBusAndSetSignal(false);
         }
+        xTaskNotify(aanderaa_controller_task_handle, AGGREGATION_TIMER_BITS, eSetBits);
```

Comment:
Whenever a sample ends we will tell the aanderaaController to compute mean/stdev.

Reply:
This used to be done by a software timer with a period of "currentAggPeriodMin", but the time that the aanderaa controller computes its means/stdev needs to be tied to the end of a sample, which is done here by notifying the aanderaaController app that it is time to compute its means/stdevs.